src/care/KeyValueSorter_impl.h

```diff
@@ -55,10 +57,12 @@ namespace care {
 /// @return void
 ///////////////////////////////////////////////////////////////////////////
 template <typename KeyT, typename ValueT, typename Exec>
-CARE_INLINE void sortKeyValueArrays(host_device_ptr<KeyT> & keys,
-                                    host_device_ptr<ValueT> & values,
-                                    const size_t start, const size_t len,
-                                    const bool noCopy)
+CARE_INLINE
+std::enable_if_t<std::is_arithmetic<typename CHAIDataGetter<T, RAJADeviceExec>::raw_type>::value, void>
```

Comment:
This needs to use one of the templated types, likely `KeyT` or `ValueT`.
```suggestion
std::enable_if_t<std::is_arithmetic<typename CHAIDataGetter<KeyT, RAJADeviceExec>::raw_type>::value, void>
```

Reply:
Fixed.